index.css

```diff
@@ -1,15 +1,4 @@
-h1 {
-    color: green;
-}
-
 body {
-    margin: 2rem;
-}
-h2, h3 {
-    display: inline;
-}
-
-.h2-heading {
-    text-transform: capitalize;
-    display: block;
+    background-color: burlywood;
+    margin-left: 2rem;
 }
```

Comment:
You've removed everyone's style but the idea was to build from what others started.

Reply:
I wrongly accepted the changes and now  i fixed the changes and it is updated as you can see